sympy/physics/vector/point.py

```diff
@@ -90,12 +90,14 @@ def _pdict_list(self, other, num):
                          ' and ' + self.name)
 
     def a1pt_theory(self, otherpoint, outframe, interframe):
-        """Sets the acceleration of this point with the 1-point theory.
+        r"""Sets the acceleration of this point with the 1-point theory.
 
         The 1-point theory for point acceleration looks like this:
 
-        ^N a^P = ^B a^P + ^N a^O + ^N alpha^B x r^OP + ^N omega^B x (^N omega^B
-        x r^OP) + 2 ^N omega^B x ^B v^P
+        .. math::
+            {}^Na^P = {}^Ba^P + {}^N a^O + {}^N\alpha^B \times r^{OP}
+                    + {}^N\omega^B \times ({}^N\omega^B \times r^{OP})
+                    + 2 {}^N\omega^B \times {}^Bv^P
```

Comment:
@moorepants, would you be ok with


```suggestion
        .. for console reading
            ^N a^P = ^B a^P + ^N a^O + ^N alpha^B x r^OP + ^N omega^B x (^N omega^B
            x r^OP) + 2 ^N omega^B x ^B v^P

        .. math::
            {}^Na^P = {}^Ba^P + {}^N a^O + {}^N\alpha^B \times r^{OP}
                    + {}^N\omega^B \times ({}^N\omega^B \times r^{OP})
                    + 2 {}^N\omega^B \times {}^Bv^P
```

The `.. for console reading` makes it not appear in the sphinx docs at all, while still being visible first in the plaintext.